response_operations_ui/views/respondents.py

```diff
@@ -126,28 +126,33 @@ def respondent_details(respondent_id):
 @respondent_bp.route('/edit-contact-details/<respondent_id>', methods=['GET'])
 @login_required
 def view_contact_details(respondent_id):
-    respondents_details = party_controller.get_respondent_by_party_id(respondent_id)
+    respondent_details = party_controller.get_respondent_by_party_id(respondent_id)
 
-    form = EditContactDetailsForm(form=request.form, default_values=respondents_details)
+    form = EditContactDetailsForm(form=request.form, default_values=respondent_details)
 
-    return render_template('edit-contact-details.html', respondent_details=respondents_details,
-                           form=form, tab='respondents')
+    return render_template('edit-contact-details.html', respondent_details=respondent_details, form=form,
+                           tab='respondents', respondent_id=respondent_id)
 
 
 @respondent_bp.route('/edit-contact-details/<respondent_id>', methods=['POST'])
 @login_required
 def edit_contact_details(respondent_id):
-    form = request.form
-    contact_details_changed = party_controller.update_contact_details(respondent_id, form)
+    form = EditContactDetailsForm(form=request.form)
+    if not form.validate():
+        logger.info('Failed validation')
+        contact_details = party_controller.get_respondent_by_party_id(respondent_id)
+
+        return render_template('edit-contact-details.html', form=form, tab='respondents',
+                               respondent_id=respondent_id, errors=form.errors,
+                               respondent_details=contact_details)
 
-    if 'emailAddress' in contact_details_changed:
-        flash(f'Contact details changed and verification email sent to {form.get("email")}')
-    elif len(contact_details_changed) > 0:
-        flash('Contact details changed')
     else:
-        flash('No updates were necessary')
+        logger.info('Updating details')
```

Comment:
Could we add more information to this log line, eg `logger.info('Updating respondent details', respondent_id=respondent_id)`